vpr/src/place/place_delay_model.h

```diff
@@ -119,7 +119,7 @@ class OverrideDelayModel : public PlaceDelayModel {
         const t_placer_opts& placer_opts,
         const t_router_opts& router_opts,
         int longest_length) override;
-    float delay(int from_x, int from_y, int from_pin, int to_x, int to_y, int to_pin) const override;
+    float delay(int from_x, int from_y, int from_pin, int to_x, int to_y, int to_pin, int layer_num) const override;
```

Comment:
Comment routines you know.  E.g.: delay: returns delay from the specified (x,y) to the specified (x,y) with both endpoints on layer_num and the specified from and to pins
(from and to pins are assumed to be on the most common type of block?)